### ISSUE TITLE
There might be an error in Topic 5, Part 1: Bagging 4. Out-of-Bag Error subheading

### ISSUE THREAD

**Issue by fatih-boyar** [Yorko/mlcourse.ai#753]
[link_to_notebook](https://github.com/Yorko/mlcourse.ai/blob/main/jupyter_english/topic05_ensembles_random_forests/topic5_part1_bagging.ipynb)
In Topic 5, Part 1: Bagging and under the heading 4. Out-of-Bag Error, It starts as:

_> Looking ahead, in case of Random Forest, there is no need to use cross-validation or hold-out samples in order to get an unbiased error estimation. Why? Because, in ensemble techniques, the error estimation takes place internally._

From what I understood, this must have been in the case of Bagging. Not Random Forest. Because in bagging trees, we use bootstrapping as a sampling method, approximately 1/3 of the data is not used in a single tree so this is like an internal validation set. However, Random Forest is about selecting a subset of features randomly so some of the stronger features won't dominate the model in the first splits.

If I am wrong, can you please update me? Thanks.

**Comment by Yorko** [comment]
That's the same in case of OOB estimation for Random Forest, details can be found in [sklearn docs](https://scikit-learn.org/stable/auto_examples/ensemble/plot_ensemble_oob.html)
